server/src/main/java/org/apache/iotdb/db/mpp/common/schematree/SchemaTree.java

```diff
@@ -33,22 +34,26 @@
   private SchemaNode root;
 
   /**
-   * Return all measurement paths for given path if the path is abstract and filter the result by
-   * slimit and offset. Or return the path itself.
+   * Return all measurement paths for given path pattern and filter the result by slimit and offset.
```

Comment:
`slimit`

keep the name to be consistent with the name in parameter list

Reply:
fixed